### PR TITLE
Fixed broken conditional logic on page_header_title field

### PR DESCRIPTION
**Pull request by cjg89**
**Description**

Fixed broken conditional logic on the custom Header Title Text field, which lets you override the page title text in the h1.  Previously, it would not display, even if the "Header Content - Type of Content" field was set to "default".

**Motivation and Context**

The field wasn't showing up.

**How Has This Been Tested?**

Can be verified in Dev.

**Types of changes**

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: dev/acf-export.json [hunk 1]
```diff
@@ -110,18 +110,18 @@
                 "return_format": "value"
             },
             {
-                "key": "field_58fe096728bcc",
+                "key": "field_5c5b31bf100c8",
```
Comment (jmbarne3) [inline dev/acf-export.json]: If this is the same field, we may want to update the key here so we don't lost values when the updated field is imported.

Comment (cjg89) [reply dev/acf-export.json]: The actual meta field key (`page_header_title`) is the same, so existing values should still be referenced as expected, but I'll double check this.

Comment (cjg89) [reply dev/acf-export.json]: Confirmed that deleting + re-adding the field preserves existing meta values as long as the meta key is the same 👍

Comment (jmbarne3) [reply dev/acf-export.json]: When using the `get_field` function and when ACF loads existing values into the admin screen, the `key` on the field matters. There are actually 2 database entries for every `post_meta` when using ACF. The `get_post_meta` function will only use the one entry by querying the `meta_key`, but `get_field` uses the key in someway.

**Pull request merged**